saleor/core/tasks.py

```
@@ -7,7 +7,9 @@
 from django.utils import timezone
 
 from ..celeryconf import app
-from .models import EventDelivery, EventPayload
+from .models import EventDelivery, EventPayload 
+
+from ..account.models import User 
```

Ruff

`..account.models.User` imported but unused

Remove unused import: `..account.models.User`

(F401)